lib/sycamore/sycamore/query/schema.py

```diff
@@ -21,15 +21,16 @@ def get_schema(self) -> dict[str, str]:
 
         for k, v in schema[self._index]["mappings"].items():
             if k.startswith("properties.entity") and ".keyword" not in k:
-                key = k
-                # key = k[18:]
-                result[key] = f"({type(random_sample[0]['_source']['properties']['entity'][k[18:]])}) e.g. "
+                try:
+                    key = k
+                    result[key] = f"({type(random_sample[0]['_source']['properties']['entity'][k[18:]])}) e.g. "
```

Comment:
this will break the formatting if the first "random_sample" doesn't have the key I think?